code/__DEFINES/cargo.dm

```diff
@@ -46,6 +46,8 @@
 
 #define SUPPLYPOD_X_OFFSET -16
 
+#define CARGO_CRATE_VALUE 200
```

Comment:
cargo crates are worth 500 as seen in code/modules/cargo/exports/large_objects.dm. also if you're gonna add this define, crate export value should be set to the define

Reply:
3a8fa55